daemon/graphdriver/graphtest/graphtest_unix.go

```diff
@@ -181,6 +181,12 @@ func DriverTestCreateEmpty(t *testing.T, drivername string) {
 		t.Fatal(err)
 	}
 
+	defer func() {
+		if err := driver.Remove("empty"); err != nil {
```

Comment:
It's not Base in this particular case so I think using driver directly is better